src/app/shared/service/wiki/wiki.service.ts

```diff
@@ -9,19 +9,16 @@ import { AppConfig } from '../../../app.config';
 @Injectable()
 export class WikiService {
   private gitRepoName = 'phenomenal-h2020.wiki';
-  private baseUrl = '';
-  private metadataUrl: string;
+  private baseUrl = window.location.protocol + '//';
+  private metadataUrl = '/php-phenomenal-portal-wiki';
```

Comment:
So these changes won't interfere with the what happens in this line for instance: https://github.com/phnmnl/container-phenomenal-portal/blob/master/setup_backend_host.sh#L8? Just wondering, I'm not an expert in this codebase.

Reply:
ahhh, I see how it works now... nevermind!

Reply:
👍